.github/CODEOWNERS

```diff
@@ -117,6 +117,7 @@ common/_common.scss @microsoft/cxe-red @phkuo
 ## Component packages
 packages/react-avatar/ @microsoft/cxe-red @behowell
 packages/react-menu/ @microsoft/teams-prg
+packages/react-accordion/ @microsoft/teams-prg @bsunderhus
```

Comment:
AFAIR your team agreed to not use single person owners just your team as can be seen in rest of packages